main/resources.template.tf

```diff
@@ -35,8 +35,11 @@
           {%- if resource.database_iops is defined %}
           iops = "{{resource.rds_iops}}"
           {% endif %}
-
+          {%- if resource.alias %}
```

Comment:
btw why do we need this? It will make the logic in template more complex. Why can't we keep using resource.name and backfill values correctly in the database (actually given the current logic in serializers.py it should just work since resource.name == resource.alias https://github.com/diggerhq/cli-backend/pull/653/files#diff-cf53496647845439658e076958b2b6b800a0e557b74cbbf53e44eaafc0b3dff2R777 )

Reply:
not sure what do you mean?
do you want to start using just name instead of concatenating project, environment and name, and not using alias at all?

Reply:
I mean to say that resource.name is the terraform-friendly identifier while resource.alias is the human-friendly string. It shouldn't be used in the terraform

This is the same convention we have used for project,service,env